cora/foreground/gaussianfg.py

```diff
@@ -1,14 +1,8 @@
-"""Simulate gaussian foregrounds, in particular those in the style of Santos,
-Cooray and Knox [1]_.
+"""
+Simulate gaussian foregrounds, in particular those in the style of `Santos, Cooray and Knox <http://arxiv.org/abs/astro-ph/0408515>`_.
```

Comment:
Why doesn't this work with the link in the references section? I prefer that format because it's more expected for academic stuff. Is it that you can only use references from a "Notes" section (which seems to be hinted at in the numpydoc "spec").

Reply:
They all didn't work for different reasons. Fixed now